mlir/include/mlir-c/IR.h

```diff
@@ -820,6 +831,16 @@ MLIR_CAPI_EXPORTED MlirOperation mlirBlockGetTerminator(MlirBlock block);
 MLIR_CAPI_EXPORTED void mlirBlockAppendOwnedOperation(MlirBlock block,
                                                       MlirOperation operation);
 
+/// Read the operations defined within the given buffer, containing MLIR
+/// bytecode, into the provided block.
+MLIR_CAPI_EXPORTED MlirLogicalResult mlirBlockAppendParseBytecode(
+    MlirContext context, MlirBlock block, MlirStringRef buffer);
```

Comment:
Just checking, but you know that the existing parsing mechanism can already load a bytecode buffer right? (`mlirOperationCreateParse` for example)

Reply:
Oh, my bad, totally missed that. 🤦
Thanks for the reminder, I think this PR is no longer needed and I'm going to close it.